### DISCUSSION TITLE
How do I create a navabar like below in quasasr

### DISCUSSION THREAD

**Discussion by Eyuel-Nigussie** [quasarframework/quasar#15477] [General - Components / Directives / etc]
![Screenshot from 2023-02-22 20-55-15](https://user-images.githubusercontent.com/105924760/220715517-af3cae33-27a2-4326-ac8b-d28c3ebe3e27.png)
When you click on it
![Screenshot from 2023-02-22 20-55-04](https://user-images.githubusercontent.com/105924760/220715615-adb94248-6274-4846-854b-d6a471156afc.png)

**Comment by metalsadman** [comment]
you can try to build one with this https://quasar.dev/vue-components/slide-transition#usage inside a q-menu, or q-menu might be enough https://quasar.dev/vue-components/menu#transitions
